src/Lykke.AzureStorage/Blob/AzureBlob.cs

```diff
@@ -394,5 +394,35 @@ public async Task<BlobProperties> GetPropertiesAsync(string container, string ke
 
             return blockBlob.Properties;
         }
+
+        /// <inheritdoc />
+        public async Task<string> AcquireLeaseAsync(string container, string key, TimeSpan? leaseTime, string proposedLeaseId = null)
+        {
+            var blockBlob = await GetBlockBlobReferenceAsync(container, key, createIfNotExists: true);
+
+            return await blockBlob.AcquireLeaseAsync(leaseTime, proposedLeaseId);
+        }
+
+        /// <inheritdoc />
+        public async Task ReleaseLeaseAsync(string container, string key, string leaseId)
+        {
```

Comment:
Same as above

Reply:
```container``` and ```key``` are validated in underlying methods